homeassistant/components/light/hue.py

```diff
@@ -204,7 +214,11 @@ def process_groups(hass, api, bridge, bridge_type, update_lights_cb):
             new_lights.append(bridge.lightgroups[lightgroup_id])
         else:
             bridge.lightgroups[lightgroup_id].info = info
-            bridge.lightgroups[lightgroup_id].schedule_update_ha_state()
+            dispatcher_send(
```

Comment:
hass.helpers.dispatcher.dispatcher_send(....)

Reply:
Hmm interesting, hardly any component uses this. Looks like there's some updating to do :)